src/Tickets/Commerce/Payments_Tab.php

```diff
@@ -56,42 +77,150 @@ public function register_tab() {
-			$plus_link,
-			esc_html( tribe_get_ticket_label_singular_lowercase( 'tickets_fields_settings_about_tribe_commerce' ) ),
-			$plus_link_2
-		);
-
-		$is_tickets_commerce_enabled = tec_tickets_commerce_is_enabled();
-
+        foreach ($gateways as $gateway_key => $gateway) {
+			if ( ! $gateway::should_show() ) {
+				continue;
+			}
+			$menu_html .= sprintf(
+				'<a class="%s" href="%s">%s</a>',
+				$gateway_key === $selected_section ? 'active' : '',
+				Tribe__Settings::instance()->get_url( [ 'tab' => 'payments', 'tc-section' => $gateway_key ] ),
+				$gateway->get_label()
+			);
+        }
+		if( 'main' !== $selected_section ) {
+			$current_section_key = self::$key_current_section;
+			$menu_html .= '<input type="hidden" name="' . esc_attr( $current_section_key ) . '" ' . 
+				'id="' . esc_attr( $current_section_key ) . '" value="' . esc_attr( $selected_section ) . '" />';
+		}
+		$menu_html .= '</div>';
+		
+		return [
+			'tc-section-menu' => [
+				'type' => 'html',
+				'html' => $menu_html,
+			]
+		];
+		
+	}
+	
+	/**
+	 * Method to load filters for payments tab.
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	public function filters(){
+		add_filter( 'wp_redirect', [ $this, 'filter_redirect_url' ] );
+	}
+	
+	/**
+	 * Filters the redirect URL to include section, if applicable.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $url URL of redirection.
+	 *
+	 * @return string
+	 */
+	public function filter_redirect_url( $url ) {
+		
+		// Parse URL to get query string info.
+		$url_query = wp_parse_url( $url, PHP_URL_QUERY );
+		wp_parse_str( $url_query, $args );
+		
+		// If not on the TEC Payments tab, bail.
+		if ( 
+			( empty( $args['page'] ) || Tribe__Settings::$parent_slug !== $args['page'] ) || 
+			( empty( $args['tab'] )  || self::$slug !== $args['tab'] ) 
+		) {
+			return $url;
+		}
+		
+		// If valid section not posted, bail.
+		$current_section_key = Payments_Tab::$key_current_section;
+		if ( empty( $_POST[$current_section_key] ) || 'main' === $_POST[$current_section_key] ) {
+			return $url;
+		}
+		
+		// Add section info to URL before redirecting.
+		$current_section = $_POST[$current_section_key];
+		return add_query_arg( 'tc-section', esc_attr( $current_section ), $url );
+	}
+	
+	/**
+	 * Returns the settings item for the section menu at the top of the Payments settings tab.
+	 *
+	 * @since TBD
+	 *
+	 * @return Gateway | null
+	 */
+	public function get_section_gateway(){
+		$selected_section = tribe_get_request_var( 'tc-section', 'main' );
+		return tribe( Manager::class )->get_gateway_by_key( $selected_section );
+	}
+	
+	/**
+	 * Get selected section top level menu.
+	 *
+	 * @since TBD
+	 *
+	 * @return array[]
+	 */
+	public function get_section_top_level_menu() {
+		
+		$section_gateway = $this->get_section_gateway();
+		$selected_section = tribe_get_request_var( 'tc-section', 'main' );
```

Comment:
Use the static string for the `tc-section` and avoid using `'main'` assume empty is main.